spec/index.md

```diff
@@ -2340,6 +2340,36 @@ This function returns fee percentiles, measured in millisatoshi/vbyte (1000 mill
 
 The [standard nearest-rank estimation method](https://en.wikipedia.org/wiki/Percentile#The_nearest-rank_method), inclusive, with the addition of a 0th percentile is used. Concretely, for any i from 1 to 100, the ith percentile is the fee with rank `⌈i * 100⌉`. The 0th percentile is defined as the smallest fee (excluding coinbase transactions).
 
+### IC method `fetch_logs` {#ic-fetch_logs}
+
+Given a canister ID as input provides the logs of that canister including trap messages.
+Logs are stored in a circular buffer of 4KiB.
+Log memory persists across upgrades and it is deleted if the canister is reinstalled or uninstalled.
+The log visibility is defined in `log_visibility` field of `canister_settings`.
+Users cannot call this method.
```

Comment:
The last line means that only canisters can call this method? In that case, is every canister allowed to call this method on its own canister_id?

Reply:
> The last line means that only canisters can call this method?

Yes.

> In that case, is every canister allowed to call this method on its own canister_id?

Only if the visibility is public in the current spec.

Reply:
Removed 'no user calls' restriciton.